.travis.yml

```diff
@@ -0,0 +1,4 @@
+script:
+- |
+  docker build --target test --tag georgefrancis:test0.1 .
+  docker run --mount type=bind,source="$(pwd)",target=/app --env TRELLO_KEY --env TRELLO_TKN --env TRELLO_BOARD georgefrancis:test0.1 .
```

Comment:
Is there a reason you've chosen to do this as a multiline command?

In Travis' output it's slightly harder to read, I'd recommend splitting these two commands separately, i.e.
```
script:
  - docker build ...
  - docker run ...
```

Reply:
changed